.env.development

```diff
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_BASE_URL='https://staging-api.realdevsquad.com'
-NEXT_PUBLIC_GITHUB_IMAGE_URL='https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/'
+NEXT_PUBLIC_BASE_URL='https://api.realdevsquad.com'
```

Comment:
We use `staging-api` in development environment.
Can you please revert this change back?

Reply:
Resolved.